crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -711,6 +711,7 @@ impl Analyzer<'_> {
         self.add_value(id, value);
     }
 
+    #[inline]
```

Comment:
This change looks unrelated, and probably unnecessary? I generally assume LLVM will inline a single line function.